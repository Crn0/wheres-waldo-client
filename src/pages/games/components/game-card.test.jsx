import { MemoryRouter } from 'react-router-dom';
import { describe, it, expect } from 'vitest';
import { render } from '@testing-library/react';
import Card from './game-card';
import dataTest from '../data-test';

describe('Game card', () => {
  it('renders the game image', async () => {
    const { container } = render(
      <MemoryRouter>
        <Card game={dataTest.gameList[0]} />
      </MemoryRouter>,
    );

    expect(container).toMatchSnapshot();
  });
});
