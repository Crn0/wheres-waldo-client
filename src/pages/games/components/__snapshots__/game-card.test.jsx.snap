// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Game card > renders the game image 1`] = `
<div>
  <figure>
    <a
      class=""
      href="/games/dragon-island"
    >
      <img
        alt="dragon-island"
        src="https://res.cloudinary.com/dhtzg8kkq/image/upload/v1739172513/dev/wheres_waldo/dragon-island/dragon-island-wheres-waldo_lwnw1r.webp"
      />
      <figcaption
        class="_figcaption_35bc57"
      >
        dragon-island
      </figcaption>
    </a>
  </figure>
</div>
`;
